test/upgrade/prober/configuration.go

```diff
@@ -34,17 +36,82 @@ import (
 )
 
 const (
-	configName       = "wathola-config"
-	configMountPoint = "/home/nonroot/.config/wathola"
-	configFilename   = "config.toml"
-	watholaEventNs   = "com.github.cardil.wathola"
-	healthEndpoint   = "/healthz"
+	defaultConfigName          = "wathola-config"
+	defaultConfigHomedirPath   = ".config/wathola"
+	defaultHomedir             = "/home/nonroot"
+	defaultConfigFilename      = "config.toml"
+	defaultWatholaEventsPrefix = "com.github.cardil.wathola"
+	defaultBrokerName          = "default"
+	defaultHealthEndpoint      = "/healthz"
+	defaultFinishedSleep       = 5 * time.Second
 )
 
-var (
-	eventTypes = []string{"step", "finished"}
-	brokerName = "default"
-)
+var eventTypes = []string{"step", "finished"}
+
+// Config represents a configuration for prober.
+type Config struct {
+	Wathola       WatholaConfig
+	Namespace     string
+	Interval      time.Duration
+	FinishedSleep time.Duration
+	Serving       ServingConfig
+	FailOnErrors  bool
+}
+
+// WatholaConfig represents options related strictly to wathola testing tool.
+type WatholaConfig struct {
+	Config           ConfigMapConfig
```

Comment:
Would it make sense to inline/embed this so that we don't have so many levels of inspection